packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -135,8 +135,8 @@ class ClampingScrollSimulation extends Simulation {
     @required this.velocity,
     this.friction: 0.015,
     Tolerance tolerance: Tolerance.defaultTolerance,
-  }) : super(tolerance: tolerance) {
-    assert(_flingVelocityPenetration(0.0) == _kInitialVelocityPenetration);
+  }) : assert(_flingVelocityPenetration(0.0) == _kInitialVelocityPenetration),
```

Comment:
It's ensuring that we don't regress the bug where this assert would not have been true.